src/dotnet/Chat.UI.Blazor/Pages/ChatPage.razor

```diff
@@ -4,6 +4,7 @@
 @using ActualChat.Notification.UI.Blazor
 @inherits ComputedStateComponent<ChatPage.Model>
 @layout ChatPageLayout
+@attribute [Authorize(Policy = KnownPolicies.IsUserActive)]
```

Comment:
Are we going to have authorization rules other than IsUserActive = ~UserCanAccessTheChat, etc?